cfgov/unprocessed/apps/regulations3k/css/reg-wayfinder.less

```diff
@@ -0,0 +1,64 @@
+/* ==========================================================================
+   Regulations 3000
+   Wayfinder
+   ========================================================================== */
+
+.o-regulations-wayfinder {
+    box-sizing: border-box;
+    width: 100%;
+    border-bottom: 2px solid @green;
+    position: fixed;
+    top: -80px;
```

Comment:
Turns out 80px isn't always enough to fully hide the wayfinder on small screens when the wayfinder text runs to three lines (as often happens when looking at previous versions):

![long-wayfinder-mobile](https://user-images.githubusercontent.com/1862695/56385503-4a1bf380-61ed-11e9-81a6-bb0db704a4c2.png)

So we might need to up the `top` offset or calculate it dynamically to account for longer wayfinders.

Reply:
The wayfinder is fully hiding at all screen sizes for me now.